gitops/overlays/olivier/kustomization.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: canada-dental-care-plan
+nameSuffix: -dev
+commonLabels:
+  # commonLabels must have at least one unique label
+  # per environment to ensure selectors are applied correctly
+  app.kubernetes.io/instance: dev
+labels:
+  - pairs:
+      app.kubernetes.io/part-of: canada-dental-care-plan
+      app.kubernetes.io/managed-by: teamcity
+      app.kubernetes.io/cluster: dts-dev-sced-rhp-spoke-aks
+      app.kubernetes.io/environment: dev
+      app.kubernetes.io/tier: nonprod
+resources:
+  - ../../base/frontend/
+  - ../../base/redis/
+  - ./ingresses.yaml
+patches:
+  - path: ./patches/deployments.yaml
+  - path: ./patches/services.yaml
+  - path: ./patches/stateful-sets.yaml
```

Comment:
This line targets redis, so it can be removed.

Reply:
how do you know this target redis ?

Reply:
also, removed

Reply:
> how do you know this target redis?

Because I'm the original author 😄
If you look in the `stateful-sets.yaml` file you'll see stuff about redis.